### DISCUSSION TITLE
Jetson AGX Orin and Foxglove

### DISCUSSION THREAD

**Discussion by lukashsmannheim** [foxglove/community#909] [General]
I am visualizing a rosbag on a localhost connection with foxglove. I am realizing performance issues regarding framerate and location precision of e.g. bounding boxes.

I am running foxglove on a Jetson AGX Orin. I do not seem to have these performance issues while visualizing the same rosbag and local host connection with rviz. 

I noticed that Orin additionally "Can't Initialize nvrm channel" after launching foxglove from terminal. This does not seem to be the cause, because after launching foxglove from the desktop the same issue appears.

[Screencast from 09.01.2024 11 36 36.webm](https://github.com/foxglove/community/assets/153269349/5dc2ddcd-2c0a-47ac-a710-5d2c5f726f52)

Has anyone had similar difficulties and can help out?

**Comment by jtbandes** [comment]
How did you install foxglove – from the snap or deb package? Also, what do you see when you visit `chrome://gpu` and `https://webglreport.com/?v=2` in Chromium?
